prow/cmd/sub/main.go

```diff
@@ -86,7 +86,7 @@ func init() {
 }
 
 func main() {
-	logrusutil.ComponentInit("pubsub-subscriber")
```

Comment:
won't this end up changing the reported name from `pubsub-subscriber` to just `sub`? (maybe we don't care.)

Reply:
Yes, and ... I don't think it matters much but we could rename the component as well.